src/searchReplace.tsx

```diff
@@ -306,6 +356,32 @@ const SearchReplaceElement = (props: IProps) => {
         />
         {props.children}
       </div>
+      <div>
+        <TextField
+          appearance="outline"
+          placeholder="files to"
+          onInput={(event: any) => {
+            const excludeToggle = document
+              .getElementById('exclude-toggle')
+              ?.getAttribute('current-checked');
+            props.onFileFilter(event.target.value, excludeToggle === 'true');
```

Comment:
You should not pass the switch position here. The source of truth must be your model not the HTML element.

```suggestion
            props.onFileFilter(event.target.value);
```